Test/Machine/machine_input_tests.hpp

```diff
@@ -8,7 +8,7 @@ std::vector<netket::json> GetMachineInputs() {
   std::vector<netket::json> input_tests;
   netket::json pars;
 
-  // Ising 1d
+  /** // Ising 1d
```

Comment:
Can you also add a Tutorial file for the MPS on some model? Say AKLT, Ising or Heisenberg... ? That would be nice!